DayrSuryan/DSEthiop11.xml

```diff
@@ -452,9 +452,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   </objectDesc>
 
                   <handDesc>
-                     <handNote xml:id="h1" script="Ethiopic">
-                        <seg type="script"/>
-                     </handNote>
+                    <handNote script="Ethiopic" xml:id="h1" corresp="#p1.2 #p1.3 #a4">
+          <seg type="script">First half of the 20th-century script</seg>
+          <seg type="ink">Black, red</seg>
+          <seg type="rubrication"></seg>
```

Comment:
If nothing will be added in rubrication, the empty element can be deleted

Reply:
I prefer not to delete. Someone will want to fill it out in the future.